src/briefcase/platforms/linux/appimage.py

```diff
@@ -252,34 +252,33 @@ def verify_tools(self):
         if self.tools.host_os != "Linux":
             raise BriefcaseCommandError("AppImages can only be executed on Linux.")
 
-    def run_app(self, app: AppConfig, **kwargs):
+    def run_app(self, app: AppConfig, test_mode: bool, **kwargs):
         """Start the application.
 
         :param app: The config object for the app
+        :param test_mode: Boolean; Is the app running in test mode?
         """
-        self.logger.info("Starting app...", prefix=app.app_name)
         try:
+            # Set up the log stream
+            kwargs = self._prepare_log_stream(app=app, test_mode=test_mode)
+
             # Start the app in a way that lets us stream the logs
             log_popen = self.tools.subprocess.Popen(
```

Comment:
To help draw the difference from macOS, I'd rename all these `Popen` objects from `log_popen` to `app_popen`. It gets confusing to me what the `Popen` objects _really_ represent in the different codepaths.

Reply:
This same change also probably makes sense for flatpak and windows.